### PR TITLE
LPS-111634 Visually distinguish instance-wide and per-account roles

### PR DESCRIPTION
**Pull request by alee8888**
https://issues.liferay.com/browse/LPS-111634

### CODE REVIEW

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/view_account_roles.jsp [hunk 1]
```diff
@@ -67,6 +67,13 @@ renderResponse.setTitle(accountEntryDisplay.getName());
 					value="<%= accountRole.getDescription(locale) %>"
 				/>
 
+				<liferay-ui:search-container-column-text
+					cssClass="table-cell-expand-small table-cell-minw-150"
+					href="<%= rowURL %>"
+					name="type"
+					value='<%= (accountRole.getAccountEntryId() != 0) ? LanguageUtil.get(request, "owned") : LanguageUtil.get(request, "shared") %>'
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/view_account_roles.jsp:74]: What if we encapsulated this logic in a method in `AccountRoleDisplay`? That way all the JSP has to do is ask for the label

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@alee8888 it's getting close, but I think we can clean it up even more. Let's move as much logic as possible into the java class.

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/view_account_roles.jsp (hunk 1 above)
Comment (alee8888) [reply modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/view_account_roles.jsp:74]: Good idea Drew, thank you.  I should try to move as much Java logic out of JSP as possible. Will resend.

**Comment by alee8888** [comment]
ci:stop

**Comment by alee8888** [comment]
ci:test:sf

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/AccountRoleDisplay.java [hunk 2]
```diff
@@ -34,6 +34,10 @@ public static AccountRoleDisplay of(AccountRole accountRole)
 		return new AccountRoleDisplay(accountRole);
 	}
 
+	public long getAccountEntryId() {
+		return _accountRole.getAccountEntryId();
+	}
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/AccountRoleDisplay.java:40]: What do you need this method for?

Comment (alee8888) [reply modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/AccountRoleDisplay.java]: Apologies, it is no longer needed. I should have further cleaned up the jsp/method I created.

**Comment by alee8888** [comment]
ci:test:sf

**Comment by alee8888** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
ci:test:relevant

File: portal-impl/src/content/Language.properties [hunk 3]
```diff
@@ -3330,6 +3330,7 @@ overline=Overline
 override-inherited-restrictions=Override Inherited Restrictions
 overwrite-the-existing-attachment-with-the-removed-one=Overwrite the existing attachment with the removed one.
 overwrite-the-existing-entry-with-the-one-from-the-recycle-bin=Overwrite the existing entry with the one from the Recycle Bin.
+owned=Owned
```
Comment (drewbrokke) [inline portal-impl/src/content/Language.properties:3333]: @alee8888 let's move this change to its own separate commit.

**Review by drewbrokke — CHANGES_REQUESTED** [review]

**Comment by Sharryxue** [comment]
No related failure found in ci:test:relevant suite.

**Comment by alee8888** [comment]
ci:test:sf

File: portal-impl/src/content/Language.properties (hunk 3 above)
Comment (alee8888) [reply portal-impl/src/content/Language.properties:3333]: Moved into its own separate commit and force pushed the changes.

**Review by drewbrokke — APPROVED** [review]

**Comment by drewbrokke** [comment]
ci:forward

**Comment by drewbrokke** [comment]
@alee8888 can you please rebase and resend?

**Pull request closed without merging**